web/templates/stream_sidebar_row.hbs

```diff
@@ -8,9 +8,10 @@
                 {{> stream_privacy }}
             </span>
 
-            <a href="{{url}}" title="{{name}}" class="stream-name">{{name}}</a>
+            <span title="{{name}}" class="stream-name">{{name}}</span>
 
             <div class="stream-markers-and-controls">
+                <a href="{{url}}" id="stream-feed" data-tooltip-template-id="stream-feed-template" class="fa fa-align-right goto-interleaved-stream-view"></a>
```

Comment:
Oh, I see, you didn't understand this part:

> > I am not sure @alya what do you mean this in #30565:

> Do not show the channel feed button in the left sidebar, regardless of how this option is configured.

What was meant was "The new design does not call for this channel feed button" -- so we should delete this line and the associated tooltip code, in favor of an item in the stream's popover menu that's present if your setting is such that the channel feed isn't the default click action.

Reply:
Done.

<img width="503" alt="Screenshot 2024-07-08 at 3 40 46 PM" src="https://github.com/zulip/zulip/assets/25124304/1aced5e1-f6a1-4c08-9e38-0cc245e11d18">